### PR TITLE
Super admin

### PR DESCRIPTION
**Pull request by princeBacancy**

### CODE REVIEW

File: app/views/layouts/partials/_super_admin_dashboard.html.erb [hunk 1]
```diff
@@ -59,6 +59,21 @@
...
+        <div id="refresh">
+          <li class="nav-item avatar dropdown">
+            <a class="nav-link dropdown-toggle waves-effect waves-light" id="navbarDropdownMenuLink-5" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+              <span class="badge badge-danger ml-2"><%= get_notifications.count %></span><i class="fa fa-bell" aria-hidden="true"></i>
+            </a>
+            <div class="dropdown-menu dropdown-menu-lg-right dropdown-secondary" aria-labelledby="navbarDropdownMenuLink-5">
+              <% if @notifications.count > 0 %>  
+                <a style="color:red" class="dropdown-item waves-effect waves-light" id="mark_as_read" href="#">mark all as read</a>
+              <% end %>
+              <% @notifications.each do |notification| %>
```
Comment (pinkal-bacancy) [inline app/views/layouts/partials/_super_admin_dashboard.html.erb:71]: Move this code inside if condition.

File: config/routes.rb [hunk 2]
```diff
@@ -27,9 +29,11 @@
 
   resources :medico do
     collection do
+      get :get_current_user_id
```
Comment (pinkal-bacancy) [inline config/routes.rb:32]: what is the use of this action?

File: app/views/layouts/partials/_super_admin_dashboard.html.erb (hunk 1 above)
Comment (princeBacancy) [reply app/views/layouts/partials/_super_admin_dashboard.html.erb:71]: Ok

File: config/routes.rb (hunk 2 above)
Comment (princeBacancy) [reply config/routes.rb:32]: This action is callles from channel js to get current user id there

**Pull request merged**